modules/core/src/test/scala/com.snowplowanalytics.snowplow.bigquery/processing/ProcessingSpec.scala

```diff
@@ -259,8 +274,9 @@ class ProcessingSpec extends Specification with CatsEffect {
     else TestControl.executeEmbed(io)
   }
 
-  def alter2        = alter2_base(legacyColumns = false, timeout = false)
-  def alter2_legacy = alter2_base(legacyColumns = true, timeout = true)
+  def alter2             = alter2_base(legacyColumns = false, timeout = false, legacyColumnMode = false)
+  def alter2_legacy      = alter2_base(legacyColumns = true, timeout = true, legacyColumnMode = false)
+  def alter2_full_legacy = alter2_base(legacyColumns = true, timeout = true, legacyColumnMode = true)
```

Comment:
Same as above.  `alter2_legacy` and `alter2_full_legacy` test exactly the same thing twice.